### PR TITLE
feat(email): Update verification email for OAuth reliers.

### LINKED ISSUE #2859: Update verification email for non-Sync FxA reliers
Current:
image

Proposed:
image

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Show the relier name in the email in the header and in
the main text.

fixes #2859 

screenshot 2019-02-08 at 12 02 59

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r?

File: lib/senders/email.js [hunk 1]
```diff
@@ -578,7 +583,8 @@ module.exports = function (log, config, oauthdb) {
         location: this._constructLocationString(message),
         oneClickLink: links.oneClickLink,
         privacyUrl: links.privacyUrl,
-        sync: message.service,
+        serviceName: serviceName,
+        sync: message.service === 'sync',
```
Comment (vbudhram) [inline lib/senders/email.js]: I don't see any usage of `sync` in the email template, is it needed?

File: test/local/senders/email.js [hunk 2]
```diff
@@ -702,7 +716,31 @@ describe(
...
+              })
+              message.service = 'foo'
+              return mailer[type](message)
+            }
+          )
+          it(
+            'works without a service',
+            () => {
+              mailer.mailer.sendMail = stubSendMail(emailConfig => {
+                assert.isFalse(oauthClientInfo.fetch.called)
```
Comment (vbudhram) [inline test/local/senders/email.js]: You could assert that the email doesn't contain `and continue to` string.

File: lib/senders/email.js (hunk 1 above)
Comment (shane-tomlinson) [reply lib/senders/email.js]: It's used in the built template [here](https://github.com/mozilla/fxa-auth-server/blob/ae2b64198443c9f220f88f0aa18317c367565e49/lib/senders/templates/verify.html#L14).

**Pull request merged**